website/src/index-page.tsx

```diff
@@ -1,119 +1,397 @@
-    },
-  },
+const gradients: [string, string][] = [
+  ['#8b5cf6', '#6d28d9'], // violet
+  ['#06b6d4', '#0e7490'], // cyan
+  ['#f59e0b', '#d97706'], // amber
 ]
-import Link from 'next/link'
 
-export default function IndexPage() {
+function pickGradient(i: number) {
+  const gradient = gradients[i % gradients.length]
+  if (!gradient) {
+    throw new Error('No gradient found')
+  }
+  return gradient
+}
+
+export function IndexPage(): ReactElement {
   return (
-    <>
-      <HeroGradient
-        title="Time to Relax with GraphQL Yoga"
-        description="The fully-featured GraphQL Server with focus on easy setup, performance and great developer experience."
-        link={[
-          {
-            href: '/docs',
-            children: 'Read the docs',
-            title: 'Read the Yoga Docs',
-          },
-          {
-            href: '/tutorial/basic',
-            children: 'Start the Tutorial',
-            title: 'Start the Tutorial',
-            style: {
-              color: '#fff',
-              border: '1px solid #fff',
-              background: 'transparent',
+    <div className="flex flex-col">
+      <FeatureWrapper>
+        <div className="w-full">
+          <div className="py-20 sm:py-24 lg:py-32 my-6">
+            <h1
+              className="max-w-screen-md mx-auto font-extrabold text-5xl sm:text-5xl lg:text-6xl text-center bg-gradient-to-r from-fuchsia-400 to-fuchsia-700 dark:from-fuchsia-400 dark:to-fuchsia-700 bg-clip-text text-transparent"
+              style={{
+                // TODO: default line height is 1, but that cuts of the "g" in the title
+                lineHeight: 1.2,
+              }}
+            >
+              GraphQL Yoga
+            </h1>
+            <p className="max-w-screen-sm mx-auto mt-6 text-2xl text-gray-600 text-center dark:text-gray-400">
+              The fully-featured GraphQL Server with focus on easy setup,
+              performance and great developer experience.
+            </p>
+            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-4">
+              <ButtonLink href="/v3">Documentation</ButtonLink>
+              <ButtonLink className="hidden lg:block" href="/tutorial/basic">
+                Tutorial
+              </ButtonLink>
+              <ButtonLink
+                className="flex flex-row gap-2 items-center"
+                href="https://github.com/dotansimha/graphql-yoga"
+              >
+                <FiGithub /> GitHub
+              </ButtonLink>
+            </div>
+          </div>
+        </div>
+      </FeatureWrapper>
+      <Feature
+        title="GraphQL over HTTP compliant"
+        description={
+          <>
+            following the{' '}
+            <TextLink
+              href="https://graphql.github.io/graphql-over-http"
+              className="text-primary-500"
+            >
+              GraphQL over HTTP specification
+            </TextLink>
+          </>
```

Comment:
I feel like this should rather be a full sentence e.g.:

```
GraphQL Yoga follows the GraphQL over HTTP specification.
```

Reply:
I am trying to avoid too much repetition with "GraphQL Yoga", the whole page is about it - I dont think we need to mention it everywhere.